setup.cfg

```diff
@@ -3,7 +3,7 @@ universal = 1
 
 [tool:pytest]
 python_files=test_*.py
-testpaths=xarray/tests
+testpaths=xarray/tests properties
```

Comment:
IIUC, do we want pytest running hypothesis tests?

Reply:
Why not? Is there a risk that they'll randomly start failing?

Note the added line to the test script that disables the whole thing if the hypothesis package is not installed.

Reply:
Also the whole test suite runs in 900ms for me

Reply:
Um, OK, then I take that back. I thought hypothesis took much longer!

Reply:
I've added ``@pytest.mark.slow``; that should settle the argument I believe?

Reply:
Great!